lib/elixir/lib/code.ex

```diff
@@ -942,6 +963,16 @@ defmodule Code do
   """
   @doc_kinds [:docs, :moduledoc, :callback_docs, :type_docs, :all]
 
+  @spec get_docs(module, :moduledoc) :: {line :: pos_integer, doc :: false | binary} | nil
+  @spec get_docs(module, :docs) :: [{function, line, kind, list, doc}] | nil
+        when function: {atom, arity}, line: pos_integer, kind: atom, doc: nil | false | binary
+  @spec get_docs(module, :callback_docs) :: [{callback, line, kind, doc}] | nil
+        when callback: {atom, arity}, line: pos_integer, kind: atom, doc: nil | false | binary
+  @spec get_docs(module, :type_docs) :: [{type, line, kind, doc}] | nil
+        when type: {atom, arity}, line: pos_integer, kind: atom, doc: nil | false | binary
+  @spec get_docs(module, :all) :: keyword | nil
+  def get_docs(module, kind)
```

Comment:
We don't need this, do we?

Reply:
We don't because we rely on the names of the arguments of the first clause to be what we want But tbh I usually avoid doing that and just define a bodyless clause so that documentatio  will be good even if we change the implementation.